templateutils/templateutils.go

```diff
@@ -90,7 +92,7 @@ func filterField(obj interface{}, field, val string, cmp func(string, string) bo
 	defer func() {
 		err := recover()
 		if err != nil {
-			fatalf("Invalid use of filter: %v", err)
+			panic(fmt.Errorf("Invalid use of filter: %v", err))
```

Comment:
Is it idiomatic in go for a package to panic instead of cleanly return an error?  In C, it's really horrible for a library to panic and result in the death of the caller.